console/billing/how-to/redeem-voucher-code.mdx

```diff
@@ -17,10 +17,10 @@ A Scaleway voucher provides you with a discount or credit to use on our services
 
 Voucher codes are unique and valid only for a defined period of time.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-  - You have a valid payment method added to your account.
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- A valid payment method added to your account.
```

Comment:
I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?